website/src/app/kb/reference/glossary/readme.mdx

```diff
@@ -0,0 +1,50 @@
+# Glossary of terms
+
+**Account Slug**: A unique identifier for your Firezone account typically
+generated automatically during sign up. This is used in the URL for your
+Firezone admin portal, e.g.
+https://app.firezone.dev/**international-widget-corporation**. You can change
+your account slug by [contacting support](mailto:support@firezone.dev).
+
+**Admin Portal**: The web-based interface where you can manage your Firezone
+account. You can access the admin portal at
+https://app.firezone.dev/**your-account-slug**.
+
+**Actor**: An Actor is a [user](/kb/deploy/users) or
+[service account](/kb/authenticate/service-accounts) that can authenticate to
+Firezone. Actors can be members of one or more [Groups](/kb/deploy/groups) which
+can be assigned one or more [Policies](/kb/deploy/policies) to grant access to
+[Resources](/kb/deploy/resources).
+
+**Gateway**: [Gateways](/kb/deploy/gateways) are Firezone servers that run on
+your infrastructure. Gateways must be defined within a Site, and any traffic
+to/from Resources associated with a Site will pass through one of that Site’s
+Gateways. We distribute the Gateway as self-contained binaries on our
+[releases page](https://github.com/firezone/firezone/releases), or as a Docker
+image or Systemd unit file with instructions shown when you deploy the Gateway
+from the admin portal. Gateways can run on everything from a Raspberry Pi to
+bare metal servers and everything in between. Gateways are designed to be
+lightweight and don't require persistent storage to function.
```

Comment:
So even if you image 100 IoT devices with the same service token, they can all run Gateways? Or do they need unique IDs?

AIUI the Gateway doesn't need to write anything to disk but does it need any unique read-only data?